DbTables/src/DbSet.cc

```diff
@@ -0,0 +1,83 @@
+
+#include "Offline/DbTables/inc/DbSet.hh"
+#include "cetlib_except/exception.h"
+#include <iomanip>
+#include <iostream>
+
+namespace mu2e {
+
+  //***********************************************
+
+void DbSet::add(int tid, int cid, DbIoV const& iov) { 
+  _emap[tid].emplace_back(cid, iov); 
```

Comment:
Is it OK to add a duplicate?  If not, should it trigger a warning or error?

Reply:
Since the lists can get long and the input should have already been checked for duplicates, I don't think it makes sense to check here.  If it is true duplicate, then it doesn't affect the existing main algorithm.  The same cid and different IoV is allowed.  Trying to imagine a use case where it is more wild-west and needs to be checked.  I think I'd add a separate duplicate check.  But I don't imagine any of this will be used by any sort of non-expert.

Reply:
Thanks for the explanation.